apps/prairielearn/src/migrations/20250606205356_variants__modified_at__nonnull_constraint_add.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE variants
+ALTER COLUMN modified_at
+SET DEFAULT CURRENT_TIMESTAMP;
```

Comment:
I believe it should be safe to add a default here, and to do so at the same time as we're adding the constraint. Past experience has made me nervous about these `NOT NULL` migrations thought, so I'm explicitly asking for more sets of eyes on this.

Reply:
🤦 🤦 🤦 

I need an AI feature that tells me to actually read the code I'm commenting on. Somehow I read the ADD CONSTRAINT line as the actual SET NOT NULL query.

Everything looks perfect! Sorry for the noise!